README.md

```diff
@@ -25,7 +25,37 @@ To execute these tests:
     For example, to create 100 notebooks in batches of 10 at a time:
 
     ```
-    ./create_notebook.sh 100 10 your_username
+    ./create_notebook.sh 100 10 your_username image_name
+    ```
+
+    To launch notebooks with GPUs, use image `minimal-gpu:2023.1`; edit the test_resources.yaml:
```

Comment:
Requiring someone to edit `test_resources.yaml` in order to run the GPU tests seems inconvenient. It would be great if we could automate things so that someone could run `create_notebook.sh --gpu ...` or something to activate these changes.

A simple solution would be to provide an alternative `test_resources.yaml` file. We could investigate solutions involving more complex templating.

I'm not suggesting we block this PR on these changes; these are more thoughts for future work.